### PR TITLE
activate octree option

### PR DESCRIPTION
**Pull request by steo85it**
3/3 (still curious to find out how to apply review suggestions...)

### CODE REVIEW

**Comment by sampotter** [comment]
Sure, NP. I added a silly little change you can try to make just to see if we can figure out how to do this...

**Comment by steo85it** [comment]
Where is the change? I can't see it this time.

File: flux/octree.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import numpy as np
+
```
[the remark this answers is not in the document]
Comment (sampotter) [reply flux/octree.py:2]: Wow, so nice! I accept!

**Comment by sampotter** [comment]
Yeah, I think that worked. I think the simple idea is like this:

1. Make a new branch for each PR
2. When you need to make a change, just commit those changes to that branch and push to your repository
3. They should populate the PR here automatically

I think...

**Comment by steo85it** [comment]
Yes, but it will still make a new commit, which is kind of "dirty"... I tried to amend the commit but it didn't work, somehow.

**Comment by steo85it** [comment]
plus, I think the initial PR was not accepted ... is that intentional?

**Comment by sampotter** [comment]
Oops, sorry! Of course, I meant to accept it...

**Comment by sampotter** [comment]
How's that?

I also "squashed" your commit. This is an option when accepting the PR. Just flattens all the commits into one. Possibly not the desired behavior for more complicated PRs, but on the other hand I believe it's good practice to keep PRs relatively small.

**Comment by steo85it** [comment]
Well, if you can "squash" them together, then it's no problem to add small modifications (at least for small, clean PRs, as you said). Perfect though, thanks!

**Comment by sampotter** [comment]
OK, great. We are learning, one small step at a time. ;-)

**Pull request merged**